pubtools/pulplib/__init__.py

```diff
@@ -16,6 +16,9 @@
     ModulemdDefaultsUnit,
     Distributor,
     PublishOptions,
+    FileSyncOptions,
+    ContainerSyncOptions,
+    YumSyncOptions,
```

Comment:
Can we list SyncOptions too please?  The reason is mainly for docs, we can't properly refer to the class in docs if it's not exported somewhere.

```suggestion
    YumSyncOptions,
    SyncOptions,
```